packages/core/src/tracing/span.ts

```diff
@@ -444,13 +444,17 @@ export class Span implements SpanInterface {
 
   /**
    * @inheritDoc
+   *
+   * @deprecated Use `spanToTraceHeader()` instead.
    */
   public toTraceparent(): string {
     return spanToTraceHeader(this);
   }
 
   /**
    * @inheritDoc
+   *
+   * @deprecated Use `toJSON()` or access the fields directly instead.
```

Comment:
```suggestion
   * @deprecated Use `spanToJSON()` or access the fields directly instead.
```

probably should direct there directly?

Reply:
good catch, thx!